### PR TITLE
Added functionality for sound volume

### LINKED ISSUE #32: Improve on the in-game sound
Every game that respects itself has an option to configure the in-game sound volume.
### ACCEPTANCE CRITERIA:

- Investigate a way to change the volume of all (or individual) in-game sounds
- Create such an option where the user is asked to input their preferred volume
- Add this new option to the settings from the main menu

### PR DESCRIPTION
**Pull request by achilleasmitos**
- Added a new function in handle_sound to handle the global sound volume
- Added a new entry to the main settings with a choice for the user to set this new function
- Created a vector for open audio files to be iterated upon by the new function
- Added the handling (add, remove) of this new vector in HandleSound function
- Renamed the files in main_settings to match our naming convention

closes #32

### CODE REVIEW

File: src/start_menu/main_settings/change_sound_volume.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include "change_sound_volume.h"
+#include "../../utils/utils.h"
+#include <iostream>
+
+void ChangeSoundVolume()
+{
+	std::cout << "Your current volume is at: " << utils::g_sound_volume << std::endl;
+	std::cout << "Please enter your desired sound volume (0-1000): ";
```
Comment (achilleasmitos) [inline src/start_menu/main_settings/change_sound_volume.cpp:8]: I am actually thinking of changing this into a 0-100 basis. It'll be a somewhat uglier code, but more intuitive to the end user. What do you think, @DIONYSUS-BASOUNAS ?

File: src/utils/handle_sound/handle_sound.h [hunk 2]
```diff
@@ -1,8 +1,14 @@
 #pragma once
 #include <string>
+#include <vector>
 
 namespace utils
 {
+// The master volume of the application. It is an integer between 0 and 1000.
+extern int g_sound_volume;
+// A list of all the open audio files.
+extern std::vector<std::string> g_audio_list;
```
Comment (achilleasmitos) [inline src/utils/handle_sound/handle_sound.h:10]: As a note, opening (and playing) an audio file does increase our program's use of memory quite a lot. I noticed the following memory usage:
a) Without any open mp3 files: ~450KB
b) With one mp3 file open: ~3000KB
c) With two mp3 files open at the same time: ~3800KB
d) When all mp3 files have been closed, the memory stays at ~2000KB

Perhaps we should consider (as a future fix) limiting the number of concurrent open audio tracks, and have this vector be a simple array of fixed length. Again, your opinion here is much appreciated, @DIONYSUS-BASOUNAS .

File: src/start_menu/main_settings/change_sound_volume.cpp (hunk 1 above)
Comment (DIONYSUS-BASOUNAS) [reply src/start_menu/main_settings/change_sound_volume.cpp:8]: yes i think it's more user-friendly to me 0-100

File: src/utils/handle_sound/handle_sound.h (hunk 2 above)
Comment (DIONYSUS-BASOUNAS) [reply src/utils/handle_sound/handle_sound.h:10]: we have to consider other sound effects we may add in the future, like a sword swing, flesh slashing, and other little sound efects that would play during the battle

**Comment by DIONYSUS-BASOUNAS** [comment]
When you finish the game the music stops playing, and even if you select to start the game again it was no music.
To get music again you have to close the game and open it again if you want music.
Please look into that.

**Review by DIONYSUS-BASOUNAS — APPROVED** [review]

**Pull request merged**